torch/csrc/distributed/c10d/ProcessGroupNCCL.cpp

```diff
@@ -1126,7 +1126,7 @@ ProcessGroupNCCL::~ProcessGroupNCCL() {
 
   // Abort communicators after all threads have exited to avoid having the
   // threads dying due to aborted communicator and raising a SIGABRT
-  std::string abortReason = c10::str("Process Group destroyed on rank ", rank_);
```

Comment:
for this file i think maybe add a log like 

`LOG(INFO) << logPrefix() << "ProcessGroupNCCL aborting communicators, check for 'abort finished' logs or look for abort hang";`

and for the reason, maybe take 'rank' and logprefix out entirely, and deal with logprefix in the LOG(INFO) call

Reply:
Well, we also want to include PG info in the abort reason. Because the NCCLUtil does not contain NCCL PG information.